crash/subsystem/storage/__init__.py

```diff
@@ -263,6 +275,49 @@ def request_age_ms(request: gdb.Value) -> int:
     """
     return jiffies_to_msec(kernel.jiffies - request['start_time'])
 
+def rq_data_dir(request: gdb.Value) -> int:
+    """
+    Returns direction of the request
+
+    This method returns 0 if the request is read and 1 if the request is write.
+
+    Args:
+        request: The ``struct request`` to query data direction in.
+
+    Returns:
+        :obj:`int`: 0 for reads, 1 for writes.
+    """
+    if request['cmd_flags'] & 1 != 0:
+        return WRITE
+    return READ
+
+def rq_is_sync(request: gdb.Value) -> bool:
+    """
+    Returns whether request is synchronous
+
+    This method returns True if the request is synchronous and False otherwise.
+
+    Args:
+        request: The ``struct request`` to query.
+
+    Returns:
+        :obj:`bool`: True for synchronous requests, False otherwise.
+    """
+    return request['cmd_flags'] & 1 == 0 or \
+           request['cmd_flags'] & (REQ_SYNC | REQ_FUA | REQ_PREFLUSH) != 0
```

Comment:
Use parens instead of \ here and in the next few stanzas.

Reply:
Aha, that's the python trick ;) Thanks.